pkg/consts/helpers.go

```diff
@@ -93,7 +93,7 @@ func extractInt32FromString(val string, businessValidator ...Int32BusinessValida
 	errKey := fmt.Errorf("%s value must be a whole number", val)
 	toInt, err := strconv.ParseInt(val, 10, 32)
 	if err != nil {
-		return nil, fmt.Errorf("error value: %w: %v", err, errKey)
```

Comment:
CodeQL suggests "fmt.Errorf call has more than one error-wrapping directive"
Is this helpful? https://stackoverflow.com/questions/65729195/wrapping-more-than-one-static-error-to-use-with-errors-is

Reply:
replace fmt.Error with sprint in lin 93.Thanks for the tips!